dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/java/org/apache/dolphinscheduler/e2e/pages/project/workflow/task/ShellTaskForm.java

```diff
@@ -39,7 +41,9 @@ public ShellTaskForm(WorkflowForm parent) {
         this.driver = parent.driver();
     }
 
+    @SneakyThrows
     public ShellTaskForm script(String script) {
+        Thread.sleep(Constants.DEFAULT_SLEEP_MILLISECONDS);
```

Comment:
Why sleep here

Reply:
If the machine configuration of CI is too low and the speed is too fast, the dom element will not be found and an error will be reported. Monaco Editor is not good at locating elements and judging rendering completion...